librz/core/disasm.c

```diff
@@ -2416,7 +2416,6 @@ static void ds_update_ref_lines(RDisasmState *ds) {
 		free(ds->prev_line_col);
 		ds->refline = strdup("");
 		ds->refline2 = strdup("");
-		ds->prev_line_col = strdup("");
```

Comment:
I'm not sure why this... Do we know?

Reply:
It is assigned NULL later in the function.

Reply:
Right!